src/pages/CreateGarden/index.tsx

```diff
@@ -38,82 +45,130 @@ export const CreateGarden = ()  => {
   const [desc, setDesc] = useState("");
   const nameChangeHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
     setName(() => e.target.value);
-  }
+  };
   const descChangeHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
     setDesc(e.target.value);
-  }
+  };
 
   const [ruleName, setRuleName] = useState("");
   const [ruleDesc, setRuleDesc] = useState("");
   const [userRules, setUserRules] = useState<NewUserRule[]>([]);
 
   const ruleNameChangeHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
     setRuleName(() => e.target.value);
-  }
+  };
   const ruleDescChangeHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
     setRuleDesc(e.target.value);
-  }
+  };
 
   const addRuleHandler = (e: React.MouseEvent<HTMLButtonElement>) => {
     const newRule = {
       name: ruleName,
       description: ruleDesc,
-    }
-    setUserRules(rules => [...rules, newRule]);
+    };
+    setUserRules((rules) => [...rules, newRule]);
     setRuleName("");
     setRuleDesc("");
-  }
+  };
 
   const createGardenHandler = (e: React.MouseEvent<HTMLButtonElement>) => {
-    console.log("Make API call to create garden here.")
-  }
+    const createGardenAndRules = async () => {
+      // TODO: FIX the endpoint
```

Comment:
Is this todo referencing the useApi that Daichi is making?

Reply:
Yup, we shouldn't call axios every time so next week can you update this?